frontend/src/components/common/ColumnSelector.js

```diff
@@ -1,10 +1,10 @@
 import React, { Component } from 'react'
-import Popover from './Popover'
+import { Popover } from '.'
```

Comment:
within the same folder I actually prefer from `./Popover` else you might have a circular dependency, right?